bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCommandWorkItem.java

```diff
@@ -212,7 +202,7 @@ private void processCommand(PullRequest pr, CensusInstance censusInstance, Path
                 } else {
                     printer.print("The command `");
                     printer.print(command.name());
-                    printer.println("` can only be used in a pull request that has not yet been integrated.");
+                    printer.println("` can not be used in pull requests. Please try to use this command on the commit.");
```

Comment:
I don't think we need to suggest using the command on a commit in the general case.

```suggestion
                    printer.println("` cannot be used in pull requests.");
```

Reply:
Fixed.